.babelrc

```diff
@@ -1,11 +1,27 @@
 {
-  "presets": [
-    "babel-preset-env",
-    "babel-preset-react"
-  ],
   "env": {
+    "test": {
+      "presets": [
+        [
+          "babel-preset-env",
+          { "modules": "umd" }
+        ]
+      ]
+    },
     "production": {
-      "presets": ["minify"]
+      "presets": [
+        [
+          "babel-preset-env",
+          { "modules": false }
```

Comment:
From [rollup.js documentation](https://rollupjs.org/guide/en#babel)

> "modules": false, otherwise Babel will convert our modules to CommonJS before Rollup gets a chance to do its thing, causing it to fail.